src/tiled/tilesetview.cpp

```diff
@@ -699,6 +699,16 @@ void TilesetView::contextMenuEvent(QContextMenuEvent *event)
             swapTilesAction->setEnabled(exactlyTwoTilesSelected);
             connect(swapTilesAction, SIGNAL(triggered()),
                     SLOT(swapTiles()));
+
+            bool onlyOneTileSelected =
+                    (selectionModel()->selectedIndexes().size() == 1);
+            // TODO: check that at least one object is selected
```

Comment:
Right. I think it's fine to leave a todo here for now, cause I'm not sure what's the best way. Maybe the context menu should be moved out of the TilesetView so that it can be separately implemented in the TilesetEditor and the TilesetDock.